### DISCUSSION TITLE
The pods are in Unrecognized status

### DISCUSSION THREAD

**Discussion by mkaramuk** [kubernetes/dashboard#9004] [Q&A]
Why a pod gets into unrecognized state? This is only happening in my GKE cluster. The pods are in Running state when they are deployed on minikube cluster. Is it a k8s dashboard bug or some misconfiguration?

GKE cluster:
![Screenshot_20240504_225328](https://github.com/kubernetes/dashboard/assets/39464118/957c3c1e-8bc7-4135-a76f-596649a67f06)

Minikube cluster:
![Screenshot_20240504_225259](https://github.com/kubernetes/dashboard/assets/39464118/ac002e06-b631-41f6-a9f1-e41d3c8e0b89)

**Comment by floreks** [comment]
You'd have to check container statuses inside a pod. I imagine there is some difference there. It could be a bug too. Can you share pod and container information here?

**Reply by mkaramuk** [reply]
Hi @floreks. Actually I don't have access that k8s cluster anymore, that's why I can't provide more information. As far as I remember the containers inside the pods are running too.

If you want you can check the application that I am trying to deploy: [https://github.com/mage-ai/helm-charts](https://github.com/mage-ai/helm-charts)
